test/utils/unit-tests-data.js

```diff
@@ -47,7 +48,8 @@ module.exports = {
         CacheControl: 'max-age=60',
         Metadata: {
             'some-key': 'some-value'
-        }
+        },
+        SSEKMSKeyId: "sse-key-arn"
```

Comment:
You should change this to single quotes for consistency with the rest of the code.